kernel/Cargo.toml

```diff
@@ -64,17 +64,18 @@ default-client = [
 ]
 
 developer-visibility = []
-simple-client = [
-  "arrow-array",
+sync-client = [
   "arrow-conversion",
+  "arrow-expression",
+  "arrow-array",
   "arrow-json",
   "arrow-select",
   "parquet"
 ]
 
 [dev-dependencies]
 arrow = { version = "^49.0", features = ["json", "prettyprint"] }
-deltakernel = { path = ".", features = ["default-client"] }
+deltakernel = { path = ".", features = ["default-client", "sync-client"] }
```

Comment:
why do we depend on ourselves here?

Reply:
it just means we require default and sync client to be enabled for running `cargo test`